src/components/activity-edit-form/ActivityEditForm.tsx

```diff
@@ -175,10 +175,10 @@ const ActivityEditForm = ({ onClose, activity }: ActivityEditFormProps) => {
     };
 
     return createPortal(
-        <div className={classes.activityEditWrapper}>
+        <div className="fullscreen-modal">
```

Comment:
For the sake of consistency, please use camelCase for class names rather than kebab-case. Additionally, I suggest refactoring the fullscreen-modal into a generic component.

Reply:
This could work, will do